### PR TITLE
Removed "..note :: Unlike the existence of the function ``public()``,…

### PR DESCRIPTION
**Pull request by jamesray1**
… there is no equivalent ``private()`` function. Variables simply default to private if initiated without the ``public()`` function."

### - Cute Animal Picture

screenshot from 2017-12-12 14-50-51

### CODE REVIEW

File: docs/viper-by-example.rst [hunk 1]
```diff
@@ -255,16 +255,10 @@ previous examples. Let's dive right in.
...
 
 Like other examples, we begin by initiating our variables - except this time,
-we're not calling them with the ``public`` function. Variables initiated this
-way are, by default, private.
-
-..note ::
-Unlike the existence of the function ``public()``, there is no equivalent
-``private()`` function. Variables simply default to private if initiated
-without the ``public()`` function.
+we're not calling them with the ``public`` function. Instead, we call them with the ``private`` function.
```
Comment (jacqueswww) [inline docs/viper-by-example.rst]: Just this one should be altered tho state that it's implicit private.

Comment (jamesray1) [reply docs/viper-by-example.rst]: OK, done.

**Comment by fubuloubu** [comment]
@jacqueswww I think the fact we re-use `public` to construct a getter for a given global is a little confusing. I wonder if there is a better or clearer term where we're not reusing something, like maybe a decorator `@getter` instead of this thing that goes around a variable's signature

**Comment by jamesray1** [comment]
From the simple open auction section:

> You may notice all of the variables being passed into the ``public``
function. By declaring the variable *public*, the variable is
callable by external contracts. Initializing the variables without the  ``public``
function defaults to a private declaration and thus only accessible to methods
within the same contract. The ``public`` function additionally creates a
‘getter’ function for the variable, accessible with a call such as
``self.get_beneficiary(some_address)``.

**Comment by DavidKnott** [comment]
LGTM, merging.

**Pull request merged**